### ISSUE TITLE
Operator enters crash loop

### ISSUE THREAD

**Issue by vkorenev** [lightbend/cloudflow#975]
**Describe the bug**
We have an EKS cluster with Cloudflow operator and 6 Cloudflow applications installed in it. I deployed another Cloudflow application to it, but it never comes up:
```
kubectl cloudflow status mw-iv-streams-app-qa1
+------------+-------------------------------------+
| Name:      |        mw-iv-streams-app-qa1        |
| Namespace: |        mw-iv-streams-app-qa1        |
| Version:   | qa1-0.1.0-23-6190da53-20210129-1237 |
| Created:   |        2021-01-29T22:04:24Z         |
| Status:    |               Unknown               |
+------------+-------------------------------------+
```
No pods were created in the `mw-iv-streams-app-qa1` namespace.
After that I noticed that the Cloudflow operator is in a crash loop:
```
kubectl -n cloudflow get pods
NAME                                 READY   STATUS             RESTARTS   AGE
cloudflow-operator-7f47d6ddc-vcqv7   0/1     CrashLoopBackOff   17         105m
```

However, if I undeploy the app, the operator stops crashing.

<details><summary>Operator log</summary>
<p>

```
22:14:21,395 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Could NOT find resource [logback-test.xml]
22:14:21,395 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Could NOT find resource [logback.groovy]
22:14:21,395 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Found resource [logback.xml] at [jar:file:/app/lib/com.lightbend.cloudflow.cloudflow-operator-2.0.21.jar!/logback.xml]
22:14:21,472 |-INFO in ch.qos.logback.core.joran.spi.ConfigurationWatchList@330222c1 - URL [jar:file:/app/lib/com.lightbend.cloudflow.cloudflow-operator-2.0.21.jar!/logback.xml] is not of type file
22:14:21,585 |-INFO in ch.qos.logback.classic.joran.action.ConfigurationAction - debug attribute not set
22:14:21,586 |-INFO in ch.qos.logback.core.joran.action.StatusListenerAction - Added status listener of type [ch.qos.logback.core.status.OnConsoleStatusListener]
22:14:21,664 |-INFO in ch.qos.logback.core.joran.action.AppenderAction - About to instantiate appender of type [ch.qos.logback.core.ConsoleAppender]
22:14:21,669 |-INFO in ch.qos.logback.core.joran.action.AppenderAction - Naming appender as [STDOUT]
22:14:21,675 |-INFO in ch.qos.logback.core.joran.action.NestedComplexPropertyIA - Assuming default type [ch.qos.logback.classic.encoder.PatternLayoutEncoder] for [encoder] property
22:14:21,694 |-INFO in ch.qos.logback.core.joran.action.AppenderAction - About to instantiate appender of type [ch.qos.logback.classic.AsyncAppender]
22:14:21,696 |-INFO in ch.qos.logback.core.joran.action.AppenderAction - Naming appender as [ASYNCSTDOUT]
22:14:21,696 |-INFO in ch.qos.logback.core.joran.action.AppenderRefAction - Attaching appender named [STDOUT] to ch.qos.logback.classic.AsyncAppender[ASYNCSTDOUT]
22:14:21,696 |-INFO in ch.qos.logback.classic.AsyncAppender[ASYNCSTDOUT] - Attaching appender named [STDOUT] to AsyncAppender.
22:14:21,696 |-INFO in ch.qos.logback.classic.AsyncAppender[ASYNCSTDOUT] - Setting discardingThreshold to 51
22:14:21,697 |-INFO in ch.qos.logback.classic.joran.action.RootLoggerAction - Setting level of ROOT logger to INFO
22:14:21,697 |-INFO in ch.qos.logback.core.joran.action.AppenderRefAction - Attaching appender named [ASYNCSTDOUT] to Logger[ROOT]
22:14:21,697 |-INFO in ch.qos.logback.classic.joran.action.LoggerAction - Setting level of logger [akka] to INFO
22:14:21,698 |-INFO in ch.qos.logback.classic.joran.action.LoggerAction - Setting level of logger [skuber.api] to WARN
22:14:21,698 |-INFO in ch.qos.logback.classic.joran.action.LoggerAction - Setting level of logger [org.apache.kafka] to WARN
22:14:21,698 |-INFO in ch.qos.logback.classic.joran.action.LoggerAction - Setting level of logger [com.zaxxer.hikari] to INFO
22:14:21,698 |-INFO in ch.qos.logback.classic.joran.action.LoggerAction - Setting level of logger [com.zaxxer.hikari.pool] to INFO
22:14:21,698 |-INFO in ch.qos.logback.classic.joran.action.LoggerAction - Setting level of logger [slick] to INFO
22:14:21,698 |-INFO in ch.qos.logback.classic.joran.action.LoggerAction - Setting level of logger [slick.basic.BasicBackend.action] to INFO
22:14:21,698 |-INFO in ch.qos.logback.classic.joran.action.LoggerAction - Setting level of logger [slick.basic.BasicBackend.stream] to INFO
22:14:21,699 |-INFO in ch.qos.logback.classic.joran.action.LoggerAction - Setting level of logger [slick.compiler] to INFO
22:14:21,699 |-INFO in ch.qos.logback.classic.joran.action.LoggerAction - Setting level of logger [slick.jdbc.DriverDataSource] to INFO
22:14:21,699 |-INFO in ch.qos.logback.classic.joran.action.LoggerAction - Setting level of logger [slick.jdbc.JdbcBackend.statement] to INFO
22:14:21,699 |-INFO in ch.qos.logback.classic.joran.action.LoggerAction - Setting level of logger [slick.jdbc.JdbcBackend.parameter] to INFO
22:14:21,699 |-INFO in ch.qos.logback.classic.joran.action.LoggerAction - Setting level of logger [slick.jdbc.JdbcBackend.benchmark] to INFO
22:14:21,764 |-INFO in ch.qos.logback.classic.joran.action.LoggerAction - Setting level of logger [slick.jdbc.StatementInvoker.result] to INFO
22:14:21,764 |-INFO in ch.qos.logback.classic.joran.action.LoggerAction - Setting level of logger [slick.jdbc.JdbcModelBuilder] to INFO
22:14:21,764 |-INFO in ch.qos.logback.classic.joran.action.ConfigurationAction - End of configuration.
22:14:21,765 |-INFO in ch.qos.logback.classic.joran.JoranConfigurator@6f106df2 - Registering current configuration as safe fallback point
2021-01-29 22:14:21,766 INFO  [Slf4jLogger] - Slf4jLogger started
2021-01-29 22:14:23,666 INFO  [ActorSystemImpl] -
Started cloudflow operator ..

+--------------------------------------------------------------------------------+
Build Info
+--------------------------------------------------------------------------------+

Name          : cloudflow-operator
Version       : 2.0.21
Scala Version : 2.13.3
sbt Version   : 1.4.5
Build Time    : 2020-12-18T12:37:31.811826Z
Build User    : runner


+--------------------------------------------------------------------------------+
JVM Resources
+--------------------------------------------------------------------------------+

Available processors    : 1
Free Memory in the JVM  : 8156792
Max Memory JVM can use  : 871759872
Total Memory in the JVM : 871759872


+--------------------------------------------------------------------------------+
GC Type
+--------------------------------------------------------------------------------+

List((Copy,java.lang:type=GarbageCollector,name=Copy), (MarkSweepCompact,java.lang:type=GarbageCollector,name=MarkSweepCompact))

+--------------------------------------------------------------------------------+
Cloudflow Context
+--------------------------------------------------------------------------------+

 pod-name:                         cloudflow-operator-7f47d6ddc-vcqv7
 pod-namespace                     cloudflow


+--------------------------------------------------------------------------------+
Deployment
+--------------------------------------------------------------------------------+

Release version : 2.0.21


2021-01-29 22:14:26,965 INFO  [ActorSystemImpl] - Bound to /0.0.0.0:5001.
2021-01-29 22:14:27,484 INFO  [ActorSystemImpl] - Connected to Kubernetes cluster: https://172.20.0.1:443
2021-01-29 22:14:33,375 INFO  [Operator] - Starting watch for skuber.CustomResource
2021-01-29 22:14:33,375 INFO  [Operator] - Getting current events for skuber.CustomResource
2021-01-29 22:14:33,378 INFO  [Operator] - Starting watch for skuber.Secret
2021-01-29 22:14:33,378 INFO  [Operator] - Getting current events for skuber.Secret
2021-01-29 22:14:33,464 INFO  [Operator] - Starting watch for skuber.Secret
2021-01-29 22:14:33,464 INFO  [Operator] - Getting current events for skuber.Secret
2021-01-29 22:14:33,568 INFO  [Operator] - Starting watch for skuber.Pod
2021-01-29 22:14:33,569 INFO  [Operator] - Getting current events for skuber.Pod
e => JsError(List((/items(0)/spec/deployments(0)/name,List(JsonValidationError(List(error.expected.jsstring),ArraySeq())))))
2021-01-29 22:14:42,467 WARN  [Operator] - Could not get current events, attempt number 0
akka.http.scaladsl.server.RejectionError: ValidationRejection({"obj.items[0].spec.deployments[0].name":[{"msg":["error.expected.jsstring"],"args":[]}]},Some(skuber.json.PlayJsonSupportForAkkaHttp$PlayJsonError: {"obj.items[0].spec.deployments[0].name":[{"msg":["error.expected.jsstring"],"args":[]}]}))
	at skuber.json.PlayJsonSupportForAkkaHttp.$anonfun$unmarshaller$1(PlayJsonSupportForAkkaHttp.scala:79)
	at play.api.libs.json.JsError.recoverTotal(JsResult.scala:93)
	at skuber.json.PlayJsonSupportForAkkaHttp.read$1(PlayJsonSupportForAkkaHttp.scala:76)
	at skuber.json.PlayJsonSupportForAkkaHttp.$anonfun$unmarshaller$2(PlayJsonSupportForAkkaHttp.scala:83)
	at akka.http.scaladsl.util.FastFuture$.$anonfun$map$1(FastFuture.scala:23)
	at akka.http.scaladsl.util.FastFuture$.strictTransform$1(FastFuture.scala:41)
	at akka.http.scaladsl.util.FastFuture$.$anonfun$transformWith$3(FastFuture.scala:51)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:447)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:56)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:93)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:94)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:93)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:48)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:48)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:175)
2021-01-29 22:14:43,968 INFO  [ConfigInputChangeEvent$] - [app: loyalty-etl-app application configuration changed (Secret config-loyalty-etl-app ADDED)]
2021-01-29 22:14:45,169 INFO  [ConfigInputChangeEvent$] - [app: loyalty-etl-uat application configuration changed (Secret config-loyalty-etl-uat ADDED)]
2021-01-29 22:14:45,666 INFO  [Operator] - Getting current events for skuber.CustomResource
e => JsError(List((/items(0)/spec/deployments(0)/name,List(JsonValidationError(List(error.expected.jsstring),ArraySeq())))))
2021-01-29 22:14:48,466 INFO  [Materializer] - [action] Element: Composite action executed in namespace Some(loyalty-etl-app):
 create-or-update
2021-01-29 22:14:48,566 WARN  [Operator] - Could not get current events, attempt number 1
akka.http.scaladsl.server.RejectionError: ValidationRejection({"obj.items[0].spec.deployments[0].name":[{"msg":["error.expected.jsstring"],"args":[]}]},Some(skuber.json.PlayJsonSupportForAkkaHttp$PlayJsonError: {"obj.items[0].spec.deployments[0].name":[{"msg":["error.expected.jsstring"],"args":[]}]}))
	at skuber.json.PlayJsonSupportForAkkaHttp.$anonfun$unmarshaller$1(PlayJsonSupportForAkkaHttp.scala:79)
	at play.api.libs.json.JsError.recoverTotal(JsResult.scala:93)
	at skuber.json.PlayJsonSupportForAkkaHttp.read$1(PlayJsonSupportForAkkaHttp.scala:76)
	at skuber.json.PlayJsonSupportForAkkaHttp.$anonfun$unmarshaller$2(PlayJsonSupportForAkkaHttp.scala:83)
	at akka.http.scaladsl.util.FastFuture$.$anonfun$map$1(FastFuture.scala:23)
	at akka.http.scaladsl.util.FastFuture$.strictTransform$1(FastFuture.scala:41)
	at akka.http.scaladsl.util.FastFuture$.$anonfun$transformWith$3(FastFuture.scala:51)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:447)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:56)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:93)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:94)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:93)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:48)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:48)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:175)
2021-01-29 22:14:48,567 INFO  [ConfigInputChangeEvent$] - [app: mw-intellicheck-reporting-dev-deploy1 application configuration changed (Secret config-mw-intellicheck-reporting-dev-deploy1 ADDED)]
2021-01-29 22:14:49,870 INFO  [Materializer] - [action] Element: Composite action executed in namespace Some(loyalty-etl-uat):
 create-or-update
2021-01-29 22:14:49,968 INFO  [ConfigInputChangeEvent$] - [app: mw-iv-streams-app-qa application configuration changed (Secret config-mw-iv-streams-app-qa ADDED)]
2021-01-29 22:14:50,968 INFO  [Materializer] - [action] Element: Composite action executed in namespace Some(mw-intellicheck-reporting-dev-deploy1):
 create-or-update
2021-01-29 22:14:51,069 INFO  [ConfigInputChangeEvent$] - [app: mw-iv-streams-app-qa application configuration changed (Secret logging ADDED)]
2021-01-29 22:14:51,672 INFO  [Operator] - Getting current events for skuber.CustomResource
2021-01-29 22:14:53,870 ERROR [ConfigInputChangeEvent$] - Detected input secret 'logging' does not have data key 'secret.conf', IGNORING configuration.
2021-01-29 22:14:53,870 INFO  [Materializer] - [action] Element: Composite action executed in namespace Some(mw-iv-streams-app-qa):
 create-or-update
create-or-update
create-or-update
create-or-update
create-or-update
create-or-update
create-or-update
create-or-update
e => JsError(List((/items(0)/spec/deployments(0)/name,List(JsonValidationError(List(error.expected.jsstring),ArraySeq())))))
2021-01-29 22:14:53,874 WARN  [Operator] - Could not get current events, attempt number 2
akka.http.scaladsl.server.RejectionError: ValidationRejection({"obj.items[0].spec.deployments[0].name":[{"msg":["error.expected.jsstring"],"args":[]}]},Some(skuber.json.PlayJsonSupportForAkkaHttp$PlayJsonError: {"obj.items[0].spec.deployments[0].name":[{"msg":["error.expected.jsstring"],"args":[]}]}))
	at skuber.json.PlayJsonSupportForAkkaHttp.$anonfun$unmarshaller$1(PlayJsonSupportForAkkaHttp.scala:79)
	at play.api.libs.json.JsError.recoverTotal(JsResult.scala:93)
	at skuber.json.PlayJsonSupportForAkkaHttp.read$1(PlayJsonSupportForAkkaHttp.scala:76)
	at skuber.json.PlayJsonSupportForAkkaHttp.$anonfun$unmarshaller$2(PlayJsonSupportForAkkaHttp.scala:83)
	at akka.http.scaladsl.util.FastFuture$.$anonfun$map$1(FastFuture.scala:23)
	at akka.http.scaladsl.util.FastFuture$.strictTransform$1(FastFuture.scala:41)
	at akka.http.scaladsl.util.FastFuture$.$anonfun$transformWith$3(FastFuture.scala:51)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:447)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:56)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:93)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:94)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:93)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:48)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:48)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:175)
2021-01-29 22:14:53,968 INFO  [ConfigInputChangeEvent$] - [app: mw-iv-streams-app-qa1 application configuration changed (Secret config-mw-iv-streams-app-qa1 ADDED)]
e => JsError(List((/spec/deployments(0)/name,List(JsonValidationError(List(error.expected.jsstring),ArraySeq())))))
2021-01-29 22:14:54,464 ERROR [Operator] - The configuration input stream failed, terminating.
akka.http.scaladsl.server.RejectionError: ValidationRejection({"obj.spec.deployments[0].name":[{"msg":["error.expected.jsstring"],"args":[]}]},Some(skuber.json.PlayJsonSupportForAkkaHttp$PlayJsonError: {"obj.spec.deployments[0].name":[{"msg":["error.expected.jsstring"],"args":[]}]}))
	at skuber.json.PlayJsonSupportForAkkaHttp.$anonfun$unmarshaller$1(PlayJsonSupportForAkkaHttp.scala:79)
	at play.api.libs.json.JsError.recoverTotal(JsResult.scala:93)
	at skuber.json.PlayJsonSupportForAkkaHttp.read$1(PlayJsonSupportForAkkaHttp.scala:76)
	at skuber.json.PlayJsonSupportForAkkaHttp.$anonfun$unmarshaller$2(PlayJsonSupportForAkkaHttp.scala:83)
	at akka.http.scaladsl.util.FastFuture$.$anonfun$map$1(FastFuture.scala:23)
	at akka.http.scaladsl.util.FastFuture$.strictTransform$1(FastFuture.scala:41)
	at akka.http.scaladsl.util.FastFuture$.$anonfun$transformWith$3(FastFuture.scala:51)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:447)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:56)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:93)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:94)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:93)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:48)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:48)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:175)
2021-01-29 22:14:54,567 ERROR [Materializer] - [action] Upstream failed.
akka.http.scaladsl.server.RejectionError: ValidationRejection({"obj.spec.deployments[0].name":[{"msg":["error.expected.jsstring"],"args":[]}]},Some(skuber.json.PlayJsonSupportForAkkaHttp$PlayJsonError: {"obj.spec.deployments[0].name":[{"msg":["error.expected.jsstring"],"args":[]}]}))
	at skuber.json.PlayJsonSupportForAkkaHttp.$anonfun$unmarshaller$1(PlayJsonSupportForAkkaHttp.scala:79)
	at play.api.libs.json.JsError.recoverTotal(JsResult.scala:93)
	at skuber.json.PlayJsonSupportForAkkaHttp.read$1(PlayJsonSupportForAkkaHttp.scala:76)
	at skuber.json.PlayJsonSupportForAkkaHttp.$anonfun$unmarshaller$2(PlayJsonSupportForAkkaHttp.scala:83)
	at akka.http.scaladsl.util.FastFuture$.$anonfun$map$1(FastFuture.scala:23)
	at akka.http.scaladsl.util.FastFuture$.strictTransform$1(FastFuture.scala:41)
	at akka.http.scaladsl.util.FastFuture$.$anonfun$transformWith$3(FastFuture.scala:51)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:447)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:56)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:93)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:94)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:93)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:48)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:48)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:175)
[WARN] [01/29/2021 22:14:55.164] [default-akka.actor.default-dispatcher-15] [default/Pool(#1->https://172.20.0.1:443)] [0 (WaitingForEndOfResponseEntity)] Ongoing request [GET /api/v1/namespaces//secrets Empty] was dropped because pool is shutting down
2021-01-29 22:14:55,166 ERROR [Operator] - The actions stream failed, terminating.
akka.stream.AbruptStageTerminationException: GraphStage [akka.stream.impl.fusing.GraphStages$IgnoreSink$$anon$10@173af8a8] terminated abruptly, caused by for example materializer or actor system termination.
2021-01-29 22:14:55,166 ERROR [Operator] - The config updates stream failed, terminating.
akka.stream.AbruptStageTerminationException: GraphStage [akka.stream.impl.fusing.GraphStages$IgnoreSink$$anon$10@aaa8374] terminated abruptly, caused by for example materializer or actor system termination.
2021-01-29 22:14:55,166 ERROR [Operator] - The status changes stream failed, terminating.
akka.stream.AbruptStageTerminationException: GraphStage [akka.stream.impl.fusing.GraphStages$IgnoreSink$$anon$10@7b347121] terminated abruptly, caused by for example materializer or actor system termination.
[WARN] [01/29/2021 22:14:55.167] [default-akka.actor.default-dispatcher-17] [default/Pool(#2->https://172.20.0.1:443)] [0 (WaitingForEndOfResponseEntity)] Ongoing request [GET /apis/cloudflow.lightbend.com/v1alpha1/namespaces//cloudflowapplications Empty] was dropped because pool is shutting down
[WARN] [01/29/2021 22:14:55.168] [default-akka.actor.default-dispatcher-14] [default/Pool(#4->https://172.20.0.1:443)] [0 (WaitingForEndOfResponseEntity)] Ongoing request [GET /api/v1/namespaces//pods Empty] was dropped because pool is shutting down
[WARN] [01/29/2021 22:14:55.578] [default-akka.actor.default-dispatcher-15] [default/Pool(shared->https://172.20.0.1:443)] [0 (WaitingForResponse)] Ongoing request [GET /apis/cloudflow.lightbend.com/v1alpha1/namespaces/mw-iv-streams-dev-app/cloudflowapplications/mw-iv-streams-dev-app Empty] was dropped because pool is shutting down
[WARN] [01/29/2021 22:14:55.581] [default-akka.actor.default-dispatcher-14] [default/Pool(shared->https://172.20.0.1:443)] [0 (WaitingForResponse)] Ongoing request [GET /apis/cloudflow.lightbend.com/v1alpha1/namespaces/mw-iv-streams-app-qa/cloudflowapplications/mw-iv-streams-app-qa Empty] was dropped because pool is shutting down
[WARN] [01/29/2021 22:14:55.582] [default-akka.actor.default-dispatcher-15] [default/Pool(shared->https://172.20.0.1:443)] [0 (WaitingForResponse)] Ongoing request [GET /api/v1/namespaces/mw-iv-streams-app-qa/secrets/chub Empty] was dropped because pool is shutting down
[WARN] [01/29/2021 22:14:55.582] [default-akka.actor.default-dispatcher-14] [default/Pool(shared->https://172.20.0.1:443)] [0 (WaitingForResponse)] Ongoing request [GET /api/v1/namespaces/mw-iv-streams-app-qa/secrets/pos Empty] was dropped because pool is shutting down
[WARN] [01/29/2021 22:14:55.582] [default-akka.actor.default-dispatcher-15] [default/Pool(shared->https://172.20.0.1:443)] [0 (WaitingForResponse)] Ongoing request [GET /api/v1/namespaces/mw-iv-streams-app-qa/secrets/dc Empty] was dropped because pool is shutting down
[WARN] [01/29/2021 22:14:55.582] [default-akka.actor.default-dispatcher-14] [default/Pool(shared->https://172.20.0.1:443)] [0 (WaitingForResponse)] Ongoing request [GET /api/v1/namespaces/mw-iv-streams-app-qa/secrets/moos Empty] was dropped because pool is shutting down
[WARN] [01/29/2021 22:14:55.583] [default-akka.actor.default-dispatcher-15] [default/Pool(shared->https://172.20.0.1:443)] [0 (WaitingForResponse)] Ongoing request [GET /api/v1/namespaces/mw-iv-streams-app-qa/secrets/iv-adjust-supply Empty] was dropped because pool is shutting down
[WARN] [01/29/2021 22:14:55.583] [default-akka.actor.default-dispatcher-14] [default/Pool(shared->https://172.20.0.1:443)] [0 (WaitingForResponse)] Ongoing request [GET /api/v1/namespaces/mw-iv-streams-app-qa/secrets/iv-sync-supply Empty] was dropped because pool is shutting down
[WARN] [01/29/2021 22:14:55.664] [default-akka.actor.default-dispatcher-14] [default/Pool(shared->https://172.20.0.1:443)] [0 (WaitingForResponse)] Ongoing request [GET /api/v1/namespaces/mw-iv-streams-app-qa/secrets/ss Empty] was dropped because pool is shutting down
2021-01-29 22:14:56,966 INFO  [Operator] - Getting current events for skuber.CustomResource
[WARN] [01/29/2021 22:15:05.276] [default-akka.actor.internal-dispatcher-28] [CoordinatedShutdown(akka://default)] Coordinated shutdown phase [actor-system-terminate] timed out after 10000 milliseconds
```
</p>
</details>

**Additional context**
- Cloudflow operator is v2.0.21.
- kubectl plugin used to deploy the last app is v2.0.22 (the download link to v2.0.21 is broken). However, I don't know what versions of kubectl plugin were used to deploy other Cloudflow apps.

According to [v2.0.22 release notes](https://github.com/lightbend/cloudflow/releases/tag/v2.0.22), mixing versions like this should be fine.

**Comment by vkorenev** [comment]
Sorry, my fault. I edited CR to replace container images and accidentally deleted one of the fields there. So the problem was in a well-formed CR json but with a missing `name` field for one of the streamlets.

**Comment by andreaTP** [comment]
> According to v2.0.22 release notes, mixing versions like this should be fine.

Correct but I have fixed the links in the 2.0.21 release anyhow, thanks for noticing!
